code/modules/mob/living/simple_animal/borer/borer_powers.dm

```diff
@@ -481,9 +497,11 @@
 		return
 
 	to_chat(src, SPAN_NOTICE("You succeed in interfacing with the host's zona bovinae, this will be a painful process for them."))
-	host.awaken_psi_basic("something in your head")
-	host.psi.psi_points = 3 /// You don't get a lot at the start.
+	host.psi = new(host)
```

Comment:
use set_psi_rank

Reply:
set_psi_rank is called in the check_psionic_trigger callback at the end of awaken_psi_basic, so I didn't call it here. 

should I still call it here? I think it gives the sound and powers early if it's called there.

Reply:
is it necessary to new the psi datum here? iirc it's newed in awaken_psi_basic if the user has no psionic

Reply:
From what I've found, yes. The check_psionic_trigger callback is proc_ref'd by psi, and if it's null the callback fails, which is why it wasn't working before